nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/test/java/org/apache/nifi/cluster/integration/Node.java

```diff
@@ -137,6 +136,19 @@ private static NodeIdentifier createNodeId() {
         return new NodeIdentifier(UUID.randomUUID().toString(), "localhost", createPort(), "localhost", createPort(), "localhost", null, null, false, null);
     }
 
+    /**
+     * Utility method which accepts {@link NiFiProperties} object but calls {@link StringEncryptor#createEncryptor(String, String, String)} with extracted properties.
+     *
+     * @param nifiProperties the NiFiProperties object
+     * @return the StringEncryptor
+     */
```

Comment:
Same comment obviously, if this same 5 lines of code are going to be copy pasted everywhere, there should be a builder / factory / utility with a default